### ISSUE TITLE
is there a way to set a keybind? 

### ISSUE THREAD

**Issue by ninrod** [gnotclub/xst#29]
I have a script that sets the font size (using kill -USR1 and xrdb to live reload the changes made to .Xresources). 

I want that when I hit shift + mousewheel-up, this script gets executed. 

Is there a way to achieve that in xst? 

thanks for the project @neeasade!

**Comment by Tmplt** [comment]
I'm guessing a bit here, but I think this can be done by appending
```c
{ Button4,      ShiftMask,      externalpipe,      {.v = "<your command here>" } },
``` 
to `mkeys[]`, alike how ALT + u is bound in `shortcuts[]`.

**Comment by ninrod** [comment]
hummm. thanks, will try that!

**Comment by niebieskitrociny** [comment]
Since there has been no movement on this, I'll close it. Feel free to reopen if you do have a followup.

**Comment by ninrod** [comment]
yes, it worked. I wish there was a way to set this externally though, without having to recompile.

**Comment by niebieskitrociny** [comment]
neeasade and I are talking about a long-ish term plan to add this functionality (along with a client-server model). I'll try and keep this issue updated when work on that happens.

**Comment by neeasade** [comment]
opening this to mention there's some steps towards xresources shortcuts in this branch: https://github.com/gnotclub/xst/tree/xresources_shortcuts
